### ISSUE TITLE
decouple in memory and on disck caches for 'mount' command

### ISSUE THREAD

**Issue by evgmik** [seaweedfs/seaweedfs#6213]
This is feature request.

Currently the amount of memory cache is hard coded within 'mount' command. Moreover it is linked to the level 1 (determined by a needle size) on disk caching, so they become essentially copies of each other.

It would be nice to configure the size of the memory cache independently from the disk size cache.

And/or do not store in the disk cache what fits in the memory cache.

**Comment by SigmaStar** [comment]
Agree, sometimes sequentially scanning files will make cache useless. how to disable it?
